packages/dev/core/src/Particles/baseParticleSystem.ts

```diff
@@ -406,6 +454,33 @@ export class BaseParticleSystem implements IClipPlanesHolder {
         );
     }
 
+    protected _setEngineBasedOnBlendMode = (blendMode: number) => {

```

Comment:
`protected _setEngineBasedOnBlendMode(blendMode: number) : void { ` for consistency